CMake parsing fails for Fetch image

```
ERROR: failure occurred when recovering node sources
Traceback (most recent call last):

  File "/home/chris/experiments/rosdiscover-evaluation/scripts/recover-node-models.py", line 377, in <module>
    main()
    └ <function main at 0x7f0aa925ce50>

  File "/home/chris/experiments/rosdiscover-evaluation/scripts/recover-node-models.py", line 371, in main
    recover_all(config)
    │           └ {'type': 'recovery', 'subject': 'fetch', 'image': 'rosdiscover-experiments/fetch:v0.8.3', 'distro': 'melodic', 'build_command...
    └ <function recover_all at 0x7f0aa925cc10>

  File "/home/chris/experiments/rosdiscover-evaluation/scripts/recover-node-models.py", line 222, in recover_all
    package_node_to_sources = obtain_node_sources(experiment_config)
                              │                   └ {'type': 'recovery', 'subject': 'fetch', 'image': 'rosdiscover-experiments/fetch:v0.8.3', 'distro': 'melodic', 'build_command...
                              └ <function obtain_node_sources at 0x7f0aa925cb80>

  File "/home/chris/experiments/rosdiscover-evaluation/scripts/recover-node-models.py", line 203, in obtain_node_sources
    generate_node_sources(experiment_config)
    │                     └ {'type': 'recovery', 'subject': 'fetch', 'image': 'rosdiscover-experiments/fetch:v0.8.3', 'distro': 'melodic', 'build_command...
    └ <function generate_node_sources at 0x7f0aa925c700>

> File "/home/chris/experiments/rosdiscover-evaluation/scripts/recover-node-models.py", line 186, in generate_node_sources
    rosdiscover.cli.main(args)
    │           │   │    └ ['sources', '--save-to', '/home/chris/experiments/rosdiscover-evaluation/experiments/recovery/subjects/fetch/rosdiscover-conf...
    │           │   └ <function main at 0x7f0aa3866b80>
    │           └ <module 'rosdiscover.cli' from '/home/chris/experiments/rosdiscover-evaluation/deps/rosdiscover/src/rosdiscover/cli.py'>
    └ <module 'rosdiscover' from '/home/chris/experiments/rosdiscover-evaluation/deps/rosdiscover/src/rosdiscover/__init__.py'>

  File "/home/chris/experiments/rosdiscover-evaluation/deps/rosdiscover/src/rosdiscover/cli.py", line 356, in main
    parsed_args.func(parsed_args)
    │           │    └ Namespace(config=<_io.TextIOWrapper name='/tmp/tmpnp2kmnsh.rosdiscover.yml' mode='r' encoding='UTF-8'>, save_to='/home/chris/...
    │           └ <function sources at 0x7f0aa38669d0>
    └ Namespace(config=<_io.TextIOWrapper name='/tmp/tmpnp2kmnsh.rosdiscover.yml' mode='r' encoding='UTF-8'>, save_to='/home/chris/...

  File "/home/chris/experiments/rosdiscover-evaluation/deps/rosdiscover/src/rosdiscover/cli.py", line 212, in sources
    config_with_sources = config.with_recovered_node_sources()
                          │      └ <function Config.with_recovered_node_sources at 0x7f0aa39ec8b0>
                          └ Config(image='rosdiscover-experiments/fetch:v0.8.3', sources=('/opt/ros/melodic/setup.bash', '/ros_ws/devel/setup.bash'), lau...

  File "/home/chris/experiments/rosdiscover-evaluation/deps/rosdiscover/src/rosdiscover/config.py", line 235, in with_recovered_node_sources
    recovered_node_sources = self.find_node_sources()
                             │    └ <function Config.find_node_sources at 0x7f0aa39ec940>
                             └ Config(image='rosdiscover-experiments/fetch:v0.8.3', sources=('/opt/ros/melodic/setup.bash', '/ros_ws/devel/setup.bash'), lau...

  File "/home/chris/experiments/rosdiscover-evaluation/deps/rosdiscover/src/rosdiscover/config.py", line 261, in find_node_sources
    for package_cmake_targets in ros.cmake_targets_for_all_packages():
                                 │   └ <function ROS1.cmake_targets_for_all_packages at 0x7f0aa5bfe040>
                                 └ <roswire.ros1.ros1.ROS1 object at 0x7f0aa3658940>

  File "/home/chris/experiments/rosdiscover-evaluation/deps/roswire/src/roswire/ros1/ros1.py", line 475, in cmake_targets_for_all_packages
    target_to_info = self.package_node_sources(package)
                     │    │                    └ ROS1Package(name='eigenpy', path='/ros_ws/src/eigenpy', messages=(), services=(), actions=())
                     │    └ <function ROS1.package_node_sources at 0x7f0aa5c34f70>
                     └ <roswire.ros1.ros1.ROS1 object at 0x7f0aa3658940>

  File "/home/chris/experiments/rosdiscover-evaluation/deps/roswire/src/roswire/ros1/ros1.py", line 456, in package_node_sources
    return self.__package_source_extractor.get_cmake_info(
           └ <roswire.ros1.ros1.ROS1 object at 0x7f0aa3658940>

  File "/home/chris/experiments/rosdiscover-evaluation/deps/roswire/src/roswire/ros1/source.py", line 37, in get_cmake_info
    return self._info_from_cmakelists(cmakelists_path, package)
           │    │                     │                └ ROS1Package(name='eigenpy', path='/ros_ws/src/eigenpy', messages=(), services=(), actions=())
           │    │                     └ '/ros_ws/src/eigenpy/CMakeLists.txt'
           │    └ <function CMakeExtractor._info_from_cmakelists at 0x7f0aa3d3d940>
           └ ROS1PackageSourceExtractor(_app_instance=AppInstance(_dockerblade=Container(daemon=DockerDaemon(url='unix:///run/user/1000/do...

  File "/home/chris/experiments/rosdiscover-evaluation/deps/roswire/src/roswire/common/source.py", line 193, in _info_from_cmakelists
    info = self._process_cmake_contents(contents, package, env)
           │    │                       │         │        └ {'CMAKE_SOURCE_DIR': './', 'CMAKE_CURRENT_SOURCE_DIR': './', 'CMAKE_CURRENT_BINARY_DIR': './', 'CMAKE_BINARY_DIR': './', 'PRO...
           │    │                       │         └ ROS1Package(name='eigenpy', path='/ros_ws/src/eigenpy', messages=(), services=(), actions=())
           │    │                       └ '#\n# Copyright (c) 2014-2019 CNRS\n# Copyright (c) 2018-2020 INRIA\n#\n\nCMAKE_MINIMUM_REQUIRED(VERSION 3.1)\n\nSET(PROJECT_...
           │    └ <function CMakeExtractor._process_cmake_contents at 0x7f0aa3d3d9d0>
           └ ROS1PackageSourceExtractor(_app_instance=AppInstance(_dockerblade=Container(daemon=DockerDaemon(url='unix:///run/user/1000/do...

  File "/home/chris/experiments/rosdiscover-evaluation/deps/roswire/src/roswire/common/source.py", line 245, in _process_cmake_contents
    properties = key_val_list_to_dict(opts.get("PROPERTIES", []))
                 │                    │    └ <method 'get' of 'dict' objects>
                 │                    └ {'PROPERTIES': ['SOVERSION']}
                 └ <function key_val_list_to_dict at 0x7f0aa43dcd30>

  File "/home/chris/experiments/rosdiscover-evaluation/deps/roswire/src/roswire/util.py", line 43, in key_val_list_to_dict
    assert len(key_values) % 2 == 0
               └ ['SOVERSION']

AssertionError: assert len(key_values) % 2 == 0
ERROR: failed to obtain node sources for experiment [/home/chris/experiments/rosdiscover-evaluation/experiments/recovery/subjects/fetch/experiment.yml]
```

--- a/src/roswire/common/nodelet_xml.py
+++ b/src/roswire/common/nodelet_xml.py
@@ -26,7 +26,7 @@ class NodeletLibrary:
 
     @property
     def entrypoint(self) -> str:
-        return self.type_ + "::OnInit"
+        return self.type_ + "::onInit"
 
 
 @attr.s(frozen=True, auto_attribs=True)

